service_container.rst

```diff
@@ -227,131 +209,182 @@ and set it on a ``$logger`` property::
-    service or by using the ``php bin/console debug:container`` console command.
+    How should you know to use ``LoggerInterface`` for the type-hint? The best way
+    is by reading the docs for whatever feature you're using. You can also use the
+    ``php bin/console debug:container`` console command to get a hint
+    to the class name for a service.
+
+Handling Multiple Services
+--------------------------
+
+Suppose you also want to email a site administrator each time a site update is
+made. To do that, you create a new class::
+
+    // src/AppBundle/Updates/SiteUpdateManager.php
+    namespace AppBundle\Updates;
+
+    use AppBundle\Service\MessageGenerator;
 
-Next, tell the container the service has a constructor argument:
+    class SiteUpdateManager
+    {
+        private $messageGenerator;
+        private $mailer;
+
+        public function __construct(MessageGenerator $messageGenerator, \Swift_Mailer $mailer)
+        {
+            $this->messageGenerator = $messageGenerator;
+            $this->mailer = $mailer;
+        }
+
+        public function notifyOfSiteUpdate()
+        {
+            $happyMessage = $this->messageGenerator->getHappyMessage();
+
+            $message = \Swift_Message::newInstance()
+                ->setSubject('Site update just happened!')
+                ->setFrom('admin@example.com')
+                ->setTo('manager@example.com')
+                ->addPart(
+                    'Someone just updated the site. We told them: '.$happyMessage
+                );
+            $this->mailer->send($message);
+
+            return $message;
+        }
+    }
+
+This uses the ``MessageGenerator`` *and* the ``Swift_Mailer`` service. To register
+this as a new service in the container, simply tell your configuration to load from
+the new ``Updates`` sub-directory:
 
 .. configuration-block::
 
     .. code-block:: yaml
 
         # app/config/services.yml
         services:
-            app.message_generator:
-                class:     AppBundle\Service\MessageGenerator
-                arguments: ['@logger']
+            # ...
+
+            # registers all classes in Services & Updates directories
+            AppBundle\:
+                resource: '../../src/AppBundle/{Service,Updates,EventDispatcher,Twig,Form}'
 
     .. code-block:: xml
 
         <!-- app/config/services.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd">
-
-            <services>
-                <service id="app.message_generator" class="AppBundle\Service\MessageGenerator">
-                    <argument type="service" id="logger" />
-                </service>
-            </services>
-        </container>
+        TODO
```

Comment:
You're welcome ;) I already did, see https://github.com/symfony/symfony-docs/pull/7857.